### ISSUE TITLE
POS Product Search Panel on Chrome: infinite loop

### ISSUE THREAD

**Issue by marcalwestf** [adempiere/adempiere-vue#1021]
#### Steps to reproduce

1. Open POS with Chrome
2. Select an Open Order
3. Click on the POS Product Search Panel (icon: magnifying glass)
4. Result: the panel shows an infinite loop, as if searching for products and not finding them.

#### Screenshot or Gif
![20210806_POS_Search_Product_Chrome_looping_forever](https://user-images.githubusercontent.com/1789408/128548429-ac02f527-922b-4cf4-82cf-ec599b96c41c.gif)

#### Expected behavior
The products should be displayed on the panel.

#### Other relevant information
- Your OS: Kububtu 20.04 
- Web Browser: Chrome Version 91.0.4472.164 (Official Build) (64-bit)
- Node.js version:
- adempiere-vue version: 5.0
- Run on local vue version, connected to https://api.erpya.com/api/adempiere/

**Comment by elsiosanchez** [comment]
Greetings! @marcalwestf you have to add the product name, code, or UPC to start searching.

**Comment by marcalwestf** [comment]
@elsiosanchez  this works OK.
The issue here is that the turning gif gives the user the impression that it is doing "something", but it is just waiting for the user to enter something.

It should display something like "Enter product name, code, or UPC" and/or not display the rotating image.
This happens on Opera and Chrome.

**Comment by elsiosanchez** [comment]
Greetings! @marcalwestf Thank you for your comments

**Comment by marcalwestf** [comment]
@elsiosanchez  I have tested more in depth this case:

1. Open the POS form immediately after starting Vue.
2. Create a new Order
3. Click on the Search Product icon (the magnifying glass)
![image](https://user-images.githubusercontent.com/1789408/129094531-51efc7a3-c81c-4a26-a82f-4bc89f3d86a3.png)
4. The _Product Search_ form pops up, with the spinner rotating indefinitely. I waited for two minutes without typing in anything and still no result.
5. Type in then any character, click anywhere else to close the form, then open the window again -> the products are displayed!!!!
6. After this, the product form displays always the last result. So it seems to be a problem at the first time the form is called after login.

It seems that the result is actually there, but the form somehow is waiting for an input to start displaying something.
Perhaps this helps.
